examples/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@ahrefs/bs-react-select-examples",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Should this version correspond with the library's version?

Reply:
I don't think it's necessary, this can go out of sync really easily. for example when patching any of libraries

Reply:
this is just an example, I don't think we need to change a version here at all.